v1/chat_messages.go

```diff
@@ -0,0 +1,7 @@
+package statusproto
+
+type ChatMessages struct {
```

Comment:
I would move it outside of `v1`. In `v1` we want to only store crucial stuff mentioned in the specification.

Reply:
Wasn't aware of that. I'll add a README.md to the folder with a simple explanation.